server/@types/adjustments/index.d.ts

```diff
@@ -181,7 +181,6 @@ export interface components {
         | 'ADDITIONAL_DAYS_AWARDED'
         | 'RESTORATION_OF_ADDITIONAL_DAYS_AWARDED'
         | 'SPECIAL_REMISSION'
-        | 'TIME_SPENT_IN_CUSTODY_ABROAD'
```

Comment:
This TIME_SPENT_IN_CUSTODY_ABROAD doesnt exist in the api so was removed when I regenerated the types via the api